package.json

```diff
@@ -43,7 +44,7 @@
   },
   "scripts": {
     "pretest": "standard",
-    "test": "nyc --cache mocha --require ./test/before.js --timeout=8000 --check-leaks",
+    "test": "nyc --cache mocha --require ./test/before.spec.js ./test/*.spec.js ./test/**/*.spec.js --timeout=8000 --check-leaks",
```

Comment:
`test/before.js` isn't a test, it's a setup file. Why were all the test files renamed, anyway? I really don't think that's appropriate for this PR.

Reply:
I added folders to the test directory to organize specs for actions and reducers into their own folders.  Adding .spec was needed to target spec files and excluding test helper files.

Reply:
I can break this out into a separate PR but I need it here for now because of the change in folder structure under the test directory.

Reply:
Reverted filename for before.spec.js to before.js.  Kept .spec.js for all spec files.